documentation/user_docs/docs/manual/Symmetrising_etc.rst

```diff
@@ -1,112 +1,267 @@
+   >> sym = SymopRotation.fold(4, [0 0 1], [0 0 0]);   % Ready to cut from a 4-fold rotationally symmetric dataset about Z
+
+
+General Transformations
+-----------------------
+
+Generalised matrix transforms are implemented as the ``SymopGeneral`` class and are defined by a 3x3 matrix and a
+3-vector. These are the transform itself and the offset. The constructor for a ``SymopGeneral`` is as follows:
+
+::
+
+   SymopGeneral(matrix, offset)
+   >> sym = SymopGeneral([0 1 0
+                          1 0 0
+                          0 0 1], [0 0 0]); % Reflection across y=x
+
+It should be noted that it is possible to get the general transformation from any of the other transformation types by
+applying the transform to the identity (for which ``R`` is a convenience property)
+
+::
+
+   >> sym = SymopRotation([0 1 0], 90, [0 0 0]);
+   >> sym.R
+
+    ans =
+
+        0.0000         0    1.0000
+             0    1.0000         0
+       -1.0000         0    0.0000
+
+   >> sym.transform_vec(eye(3))
+
+    ans =
+
+        0.0000         0    1.0000
+             0    1.0000         0
+       -1.0000         0    0.0000
+
+Groups of symmetry operators
+----------------------------
+
+For a more complex transformation involving a series of rotations and reflections it is possible to construct an array
+of transformations to be applied in sequence (as a series of pre-multiplications, i.e. applied in the reverse order of
+the list).
+
+::
+
+   % Rotate 90 deg about X, Reflect across X, Rotate back 90 deg about X
+   >> big_sym = [SymopRotation([1 0 0], 90), SymopReflection([0 1 0], [0 0 1]), SymopRotation([1 0 0], -90)];
 
 
 Commands for cuts and slices
 ============================
 
-.. _Symmetrising_etc_symmetrise_sqw:
+In Horace it is possible to symmetrise both whole S(**Q**, w)s and also to symmetrise cuts to extract data as though the
+S(**Q**, w) had been symmetrised without the overhead of transforming all pixels, only those that might contribute to
+the result.
+
+Symmetric cuts
+--------------
+
+In order to do a symmetrised cut, the ordinary ``cut`` function (see :ref:cut) is used with the appropriate symmetry
+operations additionally passed into the function as an argument after the bin axes specification (see example
+below). The ``cut`` operation will then use the symmetry operations to compute the transformations of the given
+projection, accumulate the symmetrically-related pixels into the primary binning axes and transform their pixel
+coordinates according to the symmetry operations as though the S(**Q**, w) had been wholly symmetrised.
+
+::
+
+   >> w1 = sqw(data);
+   >> sym = SymopReflection([0 1 0], [0 0 1]);
+
+   % Cut from 1 to 3 in X, accumulated with -1 to -3 reflected across the X-axis
+   >> w2 = cut(w1, ortho_proj([1 0 0], [0 1 0]), [1 0.1 3], [-inf inf], [-inf inf], [-inf inf], sym);
+   >> w3 = cut(w1, ortho_proj([1 0 0], [0 1 0]), [1 0.1 3], [1 0.1 3], [-inf inf], [-inf inf], sym)
```

Comment:
May be worth saying what ortho_proj is doing (which is here a standard XYZ projection, so doing nothing?)

For w3, you do not say what the additional cut interval in Y is doing.